typings/windows-process-tree.d.ts

```diff
@@ -15,6 +15,7 @@ declare module 'windows-process-tree' {
 		pid: number,
 		name: string,
 		memory?: number,
+		cpu?: number,
```

Comment:
I think the API should look something like this (after the TS change is merged):

```ts
export interface ProcessCpuTreeNode extends ProcessTreeNode {
  cpu: number
};
export function getProcessCpuUsage(tree: ProcessTreeNode, callback: (tree: ProcessCpuTreeNode): void;
```

That way we don't need to complicate `getProcessTree` and everything is separated nicely. Also the small amount of time needed to cross the native boundary doesn't matter since this is a long running async operation.